modules/order/src/Form/OrderForm.php

```diff
@@ -32,26 +38,44 @@ public function form(array $form, FormStateInterface $form_state) {
 
     $form['order_state'] = [
       '#type' => 'details',
-      '#title' => t('Order state'),
+      '#title' => t('Order information'),
       '#group' => 'advanced',
+      '#open' => TRUE,
       '#attributes' => [
-        'class' => ['order-form-order-state'],
-      ],
-      '#attached' => [
-        'library' => ['commerce_order/drupal.commerce_order'],
+        'class' => ['order-form-order-information'],
       ],
       '#weight' => 90,
-      '#optional' => TRUE,
     ];
 
+    if (isset($form['store_id'])) {
+      $form['store_id']['#group'] = 'order_state';
+    }
+    else {
+      // @todo: This should be hidden if only one store.
+      $form['order_state']['store'] = $this->fieldAsReadOnly(t('Store'), $order->getStore()->label());
+    }
+
     if (isset($form['state'])) {
       $form['state']['#group'] = 'order_state';
     }
+    else {
+      // @todo: We need an action button for working with states and transitions.
```

Comment:
The action button won't go in the edit form (view instead?), so we can kill the @todo for now.